Apps/Master Detective Archives RAINCODE Plus/App.json

```diff
@@ -0,0 +1,8 @@
+{
+  "Hash": "6519CCC670BF9AC0",
+  "AppId": "raincodeplus-win64-shipping.exe",
+  "AppStatus": 0,
+  "AppName": "RainCodePlus-Win64-Shipping.exe",
```

Comment:
You probably want to override this to a friendlier name.

Reply:
Done.